### ISSUE TITLE
Better error handling

### ISSUE THREAD

**Issue by karussell** [graphhopper/directions-api-java-client#35]
Instead of just "bad request" the client should print the detailed message

Current workaround something like:

System.out.println(exception.getResponseBody().toString());

**Comment by boldtrn** [comment]
This is for the swagger generated client, isn't it?

**Comment by karussell** [comment]
Yes, related/duplicate issue: https://github.com/graphhopper/directions-api-clients-route-optimization/issues/6
